package.json

```diff
@@ -19,7 +19,7 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "gherkin": "^4.0.0",
+    "gherkin": "github:cucumber/gherkin-javascript",
```

Comment:
:cry: this disallows cached installs - don't wanna hold up a patch for that, but we should at least create an issue for this if we merge it like this